wcs-importer-exporter.php

```diff
@@ -50,6 +50,9 @@ public static function init() {
 	 */
 	public static function setup_importer() {
 
+		if ( class_exists( 'WC_Subscriptions' ) && version_compare( WC_Subscriptions::$version, '2.0', '>=' ) ) {
+			add_filter('woocommerce_subscription_get_' . 'last_payment' . '_date', array(WCS_Importer_Exporter::class, 'last_payment_calculation'), 10, 3);
```

Comment:
The filter name here would be changed `'woocommerce_subscription_get_last_payment_date'`. Using `'woocommerce_subscription_get_' . 'last_payment' . '_date'` is identical to `'woocommerce_subscription_get_last_payment_date'`, only the former unnecessarily adds string concatenation. That would only be necessary if `'last_payment'` was a variable or some dynamic value. When it's static, `'woocommerce_subscription_get_last_payment_date'` is identical and a more easily understood way of writing it.